index.bs

```diff
@@ -2055,6 +2044,29 @@ The following abstract operations operate on {{ReadableStream}} instances at a h
  |startAlgorithm| throws.
 </div>
 
+<div algorithm>
+ <dfn abstract-op lt="CreateReadableByteStream"
+ id="create-readable-byte-stream">CreateReadableByteStream(|startAlgorithm|, |pullAlgorithm|,
+ |cancelAlgorithm|[, |highWaterMark|, [, |autoAllocateChunkSize|]])</dfn> performs the following
```

Comment:
Since nobody ever passes the other args, maybe omit them?

Reply:
Sure, I can remove them for now. That said, we might need to bring them back if/when we introduce "transform byte streams".